Some print functions used Python2 syntax. This fixes it to Python3 syntax.
Author: Orient Kružík (INUITS)
--- SCGid-master-orig/scgid/gct_window_plots.py	2021-10-28 06:18:47.000000000 +0200
+++ SCGid-master/scgid/gct_window_plots.py	2022-02-07 16:47:49.356047797 +0100
@@ -33,11 +33,11 @@
 axis1T = it.rfilter("coverage", axis1W)
 gcMean = axis1T.target_filter().summary_stats("gc").mean
 axis2W = calc_1d_window_asymm (axis1T, "gc", plot=False)
-print axis1W, axis2W
+print (axis1W, axis2W)
 
 figureWindow = FlexibleSelectionWindow("co2gc2")
 figureWindow.calculate(it, inc_factor=0.01, plot=True)
-print figureWindow.show()
+print (figureWindow.show())
 
 forR = (
         figureWindow.means["gc"],
@@ -62,14 +62,14 @@
     
 
 
-print "\n".join([x.ident for x in it.children])
-print [x.children for x in it.children]
+print ("\n".join([x.ident for x in it.children]))
+print ([x.children for x in it.children])
 
-print [(c.ident,c.depth) for c in it.children]
+print ([(c.ident,c.depth) for c in it.children])
 
 for c in it.children:
     for c2 in c.children:
-        print c2.ident,c2.depth
+        print (c2.ident,c2.depth)
 
 dim1axis, dim1win, points = calc_1d_window_symm(it, "coverage", plot=True)
 
@@ -78,22 +78,22 @@
 
 #calc_1d_window_symm(step1_it, "gc")
 
-print dim1axis, dim1win
-print dim2axis, dim2win
+print (dim1axis, dim1win)
+print (dim2axis, dim2win)
 
-print ""
+print ("")
 
 dim1axis, dim1win, points = calc_coverage_window_1tailed(it.df, target_taxa, plot=True)
 
 step1_it = get_window_table(it.df, dim1win, "coverage")
 dim2axis, dim2win, gc_points = calc_gc_window_1tailed(step1_it, target_taxa, plot=True)
 
-print dim1axis, dim1win
-print dim2axis, dim2win
+print (dim1axis, dim1win)
+print (dim2axis, dim2win)
 
 testWindow = FlexibleSelectionWindow("co1gc1")
 testWindow.calculate(it)
-print testWindow.show()
+print (testWindow.show())
